README.md

```diff
@@ -238,9 +240,23 @@ meaningful version, and map that to a list of available features. That enables
 it to know which polyfill or code variant to serve.
 
 Services that wish to do that using UA-CH will need to inspect the `Sec-CH-UA`
-header, that is sent by default on every request, and modify their reponse
+header, that is sent by default on every request, and modify their response
 based on that.
 
+### Browser bug workaround
+Some browser versions have well-known bugs which require content to workaround
+them. Triggering those bugs can result in browser crashes, content breakage and
+other issues, and those bugs are by definition not something that can be
+feature detected. Therefore, content needs to avoid them altogether for
+affected browser versions.
+
+For that use case, servers need to be aware of the browser and its meaningful
+version, be aware of browser bugs that impact them, and apply workarounds if
+the current browser version is impacted.
+
+Services that wish to do that using UA-CH will need to inspect the `Sec-CH-UA`
+header, sent by default on every request, and use it to modify their response.
```

Comment:
My question upon reading this is that those sorts of bugs are often fixed within a lifetime of a single "meaningful" version.

Reply:
That's not necessarily true. E.g. the recent `same-site` issues that require User-Agent sniffing to work around old versions of Safari, WebP issues in older versions of Android, etc.